x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/ExecuteStepsUpdateTask.java

```diff
@@ -33,7 +33,9 @@
 import java.util.function.LongSupplier;
 
 public class ExecuteStepsUpdateTask extends IndexLifecycleClusterStateUpdateTask {
+
     private static final Logger logger = LogManager.getLogger(ExecuteStepsUpdateTask.class);
+
```

Comment:
I think these two lines can be removed to make this file unchanged?

Reply:
74ab47138edee0dbe41843a92371c5a493fc767e